wiki/Tournaments/CWC/2018/en.md

```diff
@@ -0,0 +1,246 @@
+   - To be successfully accepted on the list, you have to ensure that you did not violate the [osu! community rules](/wiki/Rules) within the last 12 months.
+
+3. All successfully formed teams will be published after the Registration Phase.
+4. Only the 32 potentially strongest countries will participate. The potential strength of a country is determined by the online statistics of all valid candidates.
+   - If the amount of registered countries is below 32, the number might be reduced to 24, 20 or 16. The aim is always to let as many countries participate as possible!
+
+5. Mapset selectors may not participate as a player in this tournament.
+
+### Stage Instructions
+
+1. In the first stage (Group Stage), the teams will be divided into 8 groups of 4 teams.
+   - This may change depending on how many teams are accepted into the competition at the end.
+
+2. All the teams from each group will face each other.
+3. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
+   - Most matches won.
+   - Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
+   - Most beatmaps won.
+   - **Winner of the match played previously between the tied teams.**
+   - In the event of a triple tie:
+     - Have higher `∑{(total score difference) / (maximum score)}`.
+     - Winner of the rematch.
+
+4. The top 2 teams of each group will move on to the Knock-Out Stages.
+   -  This may change with the actual Group Stage setup.
+
+5. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
+6. Based on [this image](https://puu.sh/bUq5V/f1066103b0.png), the stages are split up into the following:
+
+| Stage | Match ID |
+| -----: | -------- |
+| Round of 16 |  A, B, C, D, E, F, G, H |
+| Quarterfinals | I, J, K, L & R, S, T, U |
+| Semifinals | M, N & V, W, X, Y, Z, AA |
+| Finals - Week 1 | O & AB, AC, AD |
+| Finals - Week 2 | AE, P, Q |
+
+**Winning conditions:**
+   - In Group Stage, you need to win 4 maps to win a match. (Best-of-7)
+   - In the Round of 16 and the Quarterfinals, you need to win 5 maps to win a match. (Best-of-9)
+   - In Semifinals, you need to win 6 maps to win a match. (Best-of-11)
+   - In the **Finals and Grand Finals**, you need to win 7 maps to win the match. (Best-of-13)
+
+### Match Instructions
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!catch, Team-Vs., Win Condition: 'ScoreV2'. Room name must be "CWC 2018: (TeamRed) vs (TeamBlue)".
+   - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+
+2. Players are free to select up to two warm-up maps. Using beatmaps with questionable content is prohibited. All maps must be osu!catch specific maps.
```

Comment:
Should specify about each team picks one warm-up map, rather than just being random